homeassistant/components/webmin/__init__.py

```diff
@@ -0,0 +1,30 @@
+"""The Webmin integration."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import WebminUpdateCoordinator
+
+PLATFORMS = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Webmin from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = WebminUpdateCoordinator(
+        hass, entry
+    )
+    await hass.data.setdefault(DOMAIN, {})[entry.entry_id].async_setup()
```

Comment:
```suggestion
    await hass.data[DOMAIN][entry.entry_id].async_setup()
```
No need for `setdefault` here